.travis.yml

```diff
@@ -25,7 +25,7 @@ matrix:
       language: node_js
       node_js: "8"
       jdk: oraclejdk8
-      osx_image: xcode9.3
+      osx_image: xcode9.2
```

Comment:
@DimitarTachev any particular reason why going down? instead of to latest xcode 9.4.1?

Reply:
Some of our preliminary testing discovered some issues with the v9.4 TravisCI macOS nodes configuration. Some cocoapods were throwing errors during build so we decided to not waste time investigating this now and update directly to v10.0 once it becomes publicly available. The switch back to v9.2 now is needed because TravisCI stated in their blog that v9.3 will be phased out and only v9.2 and v9.4 will be available after some time.